### PR TITLE
Use batch send instead of individual send when funding bench-tps

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

sending each transaction one-by-one is slow

#### Summary of Changes

Use batch mode to send faster

Fixes #

### CODE REVIEW

File: bench-tps/src/bench.rs [hunk 1]
```diff
@@ -607,11 +607,10 @@ impl<'a> FundingTransactions<'a> for Vec<(&'a Keypair, Transaction)> {
 
     fn send<T: BenchTpsClient>(&self, client: &Arc<T>) {
         let mut send_txs = Measure::start("send_txs");
```
Comment (ryleung-solana) [inline bench-tps/src/bench.rs:609]: Nit: is this supposed to measure only the send time or the send time + the time clone and collect the transactions?

Comment (sakridge) [reply bench-tps/src/bench.rs:609]: both I guess. re-name it `send_and_clone_txs` ?

Comment (ryleung-solana) [reply bench-tps/src/bench.rs:609]: Sounds good, please rename. Other than that, LGTM.

**Review by CriesofCarrots — DISMISSED** [review]
Since it's not really clear in the PR description, this change only affects the initial funding transfers in a bench-tps run (ie when the keys are not pre-funded). The actual bench transfers already use `send_batch`.
But lgtm, thanks @sakridge

**Pull request merged**